### PR TITLE
Finally remove list of chain ids

### PR DESCRIPTION
**Pull request by sz-piotr**
Resolves L2B-3648

### CODE REVIEW

File: packages/backend/src/modules/tvl/TvlModule.ts [hunk 1]
```diff
@@ -104,20 +105,20 @@ export function createTvlModule(
       logger,
     )
 
-  const submodules: (TvlSubmodule | undefined)[] = [
+  const submodules: TvlSubmodule[] = [
```
Comment (antooni) [inline packages/backend/src/modules/tvl/TvlModule.ts:108]: Let's for now write a simple assert that will ensure all of them are defined when the env is production or staging

File: packages/backend/src/api/controllers/status/StatusController.ts
Comment (antooni) [inline packages/backend/src/api/controllers/status/StatusController.ts:1]: methods that can be deleted:
- getPricesStatus
- getBalancesStatus
- getReportsStatus
- getAggregatedReportsStatus
- getTotalSuppliesStatus

those are no longer actively being used and can be removed, please decide whether this should be in the scope of this PR

**Review by antooni — CHANGES_REQUESTED** [review]
please resolve my comments

File: packages/backend/src/modules/tvl/TvlModule.ts (hunk 1 above)
Comment (sz-piotr) [reply packages/backend/src/modules/tvl/TvlModule.ts:108]: Why would this be in the scope of this PR? I haven't changed the behaviour of this code.

File: packages/backend/src/api/controllers/status/StatusController.ts
Comment (sz-piotr) [reply packages/backend/src/api/controllers/status/StatusController.ts:1]: I will make a new PR removing those

**Review by antooni — APPROVED** [review]

**Pull request merged**